### ISSUE TITLE
Add Beehive to awesome-go

### ISSUE THREAD

**Issue by rubiojr** [muesli/beehive#276]
Should we add Beehive to https://awesome-go.com? It's already quite popular but adding it there could make it even more visible?

@muesli: I assume that Beehive's creator would be the best person to open a PR there.

**Comment by muesli** [comment]
Good idea! I'm just not sure which category I should file it into :laughing:

Server Applications? Other Software?

**Comment by prologic** [comment]
In addition to awesome-go I would also file it under [awsome-self-hosted](https://github.com/awesome-selfhosted/awesome-selfhosted) under "Automation" :)

**Comment by muesli** [comment]
@prologic Looks like it's already there :smiley:

**Comment by prologic** [comment]
D'oh :) I forgot and didn't look! 🤣😂

**Comment by prologic** [comment]
Apparently several years ago by @isdn in PR [#1084](https://github.com/awesome-selfhosted/awesome-selfhosted/pull/1084) 🤷‍♂️

**Comment by rubiojr** [comment]
> Server Applications? Other Software?

Perhaps we could suggest the repo maintainer to open a new "Automation" category?

**Comment by muesli** [comment]
Yeah, worth a shot, will do. Or at least ask them where they would file it.
